src/coffee/mappings.coffee

```diff
@@ -43,24 +59,19 @@ class CustomFieldMappings
     }
   }
   ###
+
   mapLocalizedString: (value, typeDefinitionKey, rowIndex, langHeader, regEx = CONS.REGEX_LANGUAGE) ->
     if !regEx.test langHeader
-      @errors.push "[row #{rowIndex}:#{typeDefinitionKey}] localisedString  header '#{langHeader}' format is not valid!" unless regEx.test langHeader
+      @errors.push "[row #{rowIndex}:#{typeDefinitionKey}] localizedString  header '#{langHeader}' format is not valid!" unless regEx.test langHeader
       return
     else
       "#{langHeader}": value
 
-  mapSet: (values, typeDefinitionKey, rowIndex, elementType) ->
-    result = undefined
-    values = values.split(',')
-    result = _.map values, (value) =>
-      switch elementType.name
-        when 'Number' then @mapNumber value,typeDefinitionKey,rowIndex
-        when 'Boolean' then @mapBoolean value,typeDefinitionKey,rowIndex
-        when 'Money' then @mapMoney value,typeDefinitionKey,rowIndex
-        when 'LocalizedString' then @mapLocalizedString value, typeDefinitionKey, rowIndex
-        else value
-    _.reject(result, _.isUndefined)
+  mapString: (rawString) -> rawString
```

Comment:
Hmmmm, I see what you doing. Compare with what I did [here](https://github.com/sphereio/csv-parser-price/blob/1-implement-module/src/mapCustomFields.js#L73)